test/unit/src/common/actions/t_register-name.js

```diff
@@ -112,7 +116,7 @@ describe('register name actions', () => {
         type: ActionTypes.REGISTER_NAME,
         payload: { id: 'foo/bar', snapName: 'test-snap' }
       };
-      return store.dispatch(registerName(repository, 'test-snap'))
+      return store.dispatch(registerName(repository, 'test-snap', false))
```

Comment:
Can't it be just `false` by default?

It seems to actually be already as we only check `if (triggerBuilds)` so it would not be called for `undefined`.